.github/workflows/markdown-lint.yml

```
@@ -0,0 +1,25 @@
+name: Markdown Lint
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "**.md"
+      - "!.github/**"
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - uses: tj-actions/changed-files@v39
```

most recent bump as per ibc-go equiv workflow

```suggestion
    - uses: tj-actions/changed-files@v40
```